pyscriptjs/tests/integration/test_py_config.py

```diff
@@ -66,7 +66,7 @@ def test_py_config_external(self):
         )
         assert self.console.log.lines[-1] == "config name: app with external config"
 
-    # The default pyodide version is 0.22.1 as of writing
+    # The default pyodide version is 0.23.0 as of writing
```

Comment:
Maybe eliminate this comment to reduce number of spots we have to change the version?

Reply:
Eliminated and cleaned up the surrounding grammar a bit.